### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by brianjmurrell**
Signed-off-by: Brian J. Murrell <brian.murrell@intel.com>

### CODE REVIEW

File: mpifileutils.spec [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+BuildRequires: libcircle-mpich-devel
+Provides: mpich-daos-0
+
+%description mpich
+File utilities designed for scalability and performance.
+
+%if (0%{?suse_version} >= 1500)
+%package -n libmfu0-mpich
+Summary:	File utilities designed for scalability and performance.
+Provides: mpich-daos-0
```
Comment (daltonbohning) [inline mpifileutils.spec:117]:
```suggestion
Provides: %{name}-mpich-daos-0
```
Do the `Provides` need to explicitly prepend `name`?

Comment (brianjmurrell) [reply mpifileutils.spec:117]: Yes, you are correct there.  Thanks for the catch.

**Comment by daltonbohning** [comment]
@brianjmurrell It looks like there are some other `Provides` that also don't have `name` explicitly prepended, and it seems you did update the one that I mentioned, so I wonder if all the others need this as well? If they don't, could you explain this please?

File: mpifileutils.spec [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+BuildRequires: dtcmp-openmpi3-devel
+BuildRequires: libcircle-openmpi3-devel
+
+%description openmpi3
+File utilities designed for scalability and performance.
+
+%if (0%{?suse_version} >= 1500)
+%package -n libmfu0-openmpi3
+Summary:	File utilities designed for scalability and performance.
+Provides: openmpi3-daos-0
```
Comment (daltonbohning) [inline mpifileutils.spec:85]: `%{name}-`?

File: mpifileutils.spec [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+Development files for %{name}-openmpi3.
+%endif
+
+%if %{with_mpich}
+%package mpich
+Summary:	File utilities designed for scalability and performance.
+BuildRequires: mpich-devel
+BuildRequires: dtcmp-mpich-devel
+BuildRequires: libcircle-mpich-devel
+Provides: mpich-daos-0
```
Comment (daltonbohning) [inline mpifileutils.spec:109]:
```suggestion
Provides: %{name}-mpich-daos-0
```

File: mpifileutils.spec (hunk 2 above)
Comment (daltonbohning) [inline mpifileutils.spec:85]:
```suggestion
Provides: %{name}-openmpi3-daos-0
```

**Review by daltonbohning — COMMENTED** [review]
@brianjmurrell It looks like there are some other Provides that also don't have name explicitly prepended, and it seems you did update the one that I mentioned, so I wonder if all the others need this as well? If they don't, could you explain this please?

**Review by JohnMalmberg — APPROVED** [review]

**Pull request merged**